### DISCUSSION TITLE
Mod use statistics: account for forked mods

### DISCUSSION THREAD

**Discussion by Anixx** [ramensoftware/windhawk#167] [Ideas]
There are some mods (like "Fake high contrast") that are intended to be modified by the user (the list of processes to inject). So should not the forked mods be counted in mod use statistics?

**Comment by m417z** [comment]
The list of processes to inject can be controlled in the mod's advanced tab, there's no need to fork the mod.

Generally, Windhawk doesn't track forked mods, i.e. forked mods aren't different from manually created mods, and the relation to the origin of the fork isn't tracked.
